Halium chroot script

ZeroPointEnergy opened the pull request:
This is a simple script to directly log into a halium chroot via adb/recovery for debugging purposes. It makes it a lot less painful to read the logs or change stuff if systemd is stuck and no ssh is available yet.

ZeroPointEnergy committed:
Add halium-chroot script

Simple script that:
- Mounts rootfs.img, system.img and /target/data
- Opens an interactive shell or executes a non-interactive
  command in the chroot /target
- Umounts everything again

ZeroPointEnergy committed:
fix terminal size in chroot

z3ntu commented:
Great idea.

bhush9 commented on halium-chroot:
```diff
@@ -0,0 +1,65 @@
+then # Interactive shell
+	if [ $(which tmux) ]
+	then # with tmux installed
+		tmux new-session -s halium-adb -d 'adb shell'
```
This won't probably give me shell if I am already in the tmux session :-P

ZeroPointEnergy replied:
Well that is true :-) but without a proper terminal multiplexer it is a pain to get an interactive shell where you can inject the commands to start the chroot. Maybe I can detect an already running session and work around that.

bhush9 replied:
yeah, would be lovely.. :)

ZeroPointEnergy committed:
support existing tmux session

Create a new windows instead of a new session if the
current terminal already runs a tmux session.

bhush9 commented:
Thanks, will merge it :)
